data/yaml/joint.yaml

```diff
@@ -1053,6 +1053,8 @@ ssst:
   - W14-40
   2015:
   - W15-10
+starsem:
+  2020: [S12-1, S13-1, S14-1, S15-1, S16-2, S17-1, S18-2, S19-1]
```

Comment:
not sure why "2020"—should these be divided by year?

Reply:
Yes, careless.